spacy_transformers/layers/hf_shim.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 import srsly
 import torch
-from thinc.api import get_current_ops
+from thinc.util import get_torch_default_device
```

Comment:
Sorry if I missed that conversation, but why is this not accessible from `thinc.api`?

Reply:
Fixed.